src/osd/PrimaryLogPG.cc

```diff
@@ -6756,7 +6756,10 @@ void PrimaryLogPG::write_update_size_and_usage(object_stat_sum_t& delta_stats, o
 					       uint64_t length, bool count_bytes, bool force_changesize)
 {
   interval_set<uint64_t> ch;
-  if (length)
+  if (force_changesize) { // write_full with object size to be changed
```

Comment:
the force_write caller doesn't always set this:
<pre>
	write_update_size_and_usage(ctx->delta_stats, oi, ctx->modified_ranges,
	    0, op.extent.length, true, op.extent.length != oi.size ? true : false);
</pre>
and the name doesn't make much sense.

I think

1. we can drop count_bytes arg sinc eit is always true?
2. write_full can unconditionally pass true for force_changesize
3. we can rename force_changesize to bool writefull

Does that sound right?